apps/kyb-app/src/pages/CollectionFlow/CollectionFlow.tsx

```diff
@@ -47,6 +48,36 @@ const elems = {
   divider: Divider,
 };
 
+// TODO: Find a way to make this work via the workflow-browser-sdk `subscribe` method.
+export const useCompleteLastStep = () => {
```

Comment:
Previously the last step of the collection flow never gotten to `isCompleted: true`.